### ISSUE TITLE
Fix clipped text in footer in responsive design

### ISSUE THREAD

**Issue by atsansone** [flutter/website#10532]
From https://buganizer.corp.google.com/issues/337071311

### Observed Results

Navigating through the page having enabled the responsive design, observe that the text on the footer is clipped.

* [Screenshot macOS](https://screenshot.googleplex.com/7393wLJFJ4PqeBb.png)
* [Screenshot ChromeOS](https://screenshot.googleplex.com/86TFKHCGqsEmS6C.png)

### Expected Results

Ensure that all text on the footer should be displayed complete without clipped, cut off or disappeared.

### User Impact

All users that have the responsive design enabled, have issues because the text on the footer is not displayed properly.

### Steps to Reproduce

1. Enabled the Responsive design.
1. Open the following page: https://docs.flutter.dev/ui/widgets
1. Scrolling the page until the footer and observe how is displayed.

### GAR Assessment

[Flutter Docs](https://a11ytracker.corp.google.com/products/c2fdcf18-53d2-475d-b8b0-e5dedf8cb900?e=ProductLabels)

### GAR Details

[PW 2.13. Does your product support responsive web design such that a user will only need to scroll in one dimension to access/read all functionality and content?](go/GAR-web#responsive-design)

### Environment Details

Machine: MAC/ChromeBook
Browser: Chrome Version 120.0.6099.272 (Official Build) (64-bit)
Screen reader: NA.
Bug filling time: 15 mins.
